ESP8266/EzTimeTetrisClock_ssd1306OLED/EzTimeTetrisClock_ssd1306OLED.ino

```diff
@@ -0,0 +1,321 @@
+/*******************************************************************
+    Tetris clock that fetches its time Using the EzTimeLibrary
+
+    NOTE: THIS IS CURRENTLY CRASHING!
+
+    For use with an ESP8266
+ *                                                                 *
+    Written by Brian Lough
+    YouTube: https://www.youtube.com/brianlough
+    Tindie: https://www.tindie.com/stores/brianlough/
+    Twitter: https://twitter.com/witnessmenow
+ *******************************************************************/
+
+// ----------------------------
+// Standard Libraries - Already Installed if you have ESP32 set up
+// ----------------------------
+
+#include <SPI.h>
+#include <Wire.h>
+#include <ESP8266WiFi.h>
+
+// ----------------------------
+// Display Libraries - each one of these will need to be installed.
+// ----------------------------
+#include <Adafruit_GFX.h>
+#include <Adafruit_SSD1306.h>
```

Comment:
Can you link to the library on github and advice if it can be installed from the library manager